### PR TITLE
Update Leetcode 001

### PR DESCRIPTION
**Pull request by zmzhouXJTU**

### CODE REVIEW

File: src/001._two_sum.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        
+        Returns:
+            the indices of the two numbers from the array
+        """    
+        for i in range(len(nums)):
+            for j in range(i+1, len(nums)):
+                if nums[i] + nums[j] == target:
+                    return [i, j]
+                else:
+                    return None
```
Comment (stormsunshine) [inline src/001._two_sum.md]: The return None statement should not appear here. The function should return None only after the outer loop is completely executed.

File: src/001._two_sum.md [hunk 2]
```diff
@@ -38,9 +38,8 @@ class Solution(object):
         for i in range(len(nums)):
             for j in range(i+1, len(nums)):
                 if nums[i] + nums[j] == target:
-                    return [i, j]
-                else:
-                    return None
+			return [i, j]
+        return None  #加上这个判断会超时
```
Comment (stormsunshine) [inline src/001._two_sum.md]: If Time-Limit-Exceeded, then choose another way. The nested loops approach has time complexity O(n^2), which is indeed time-consuming. Therefore, the nested loops approach should be avoided whenever possible.

Comment (zmzhouXJTU) [reply src/001._two_sum.md]: OK， I will have a try

Comment (zmzhouXJTU) [reply src/001._two_sum.md]: @stormsunshine  I have tried to refine the code. Much thanks for reviewing the code again

File: src/001._two_sum.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        Given an array of integers, return indices of the two numbers such that they add up to a specific target.
+
+        Args:
+            nums(List[int]): the name of array of integers
+            target(int): the value that we assign
+        
+        Returns:
+            the indices of the two numbers from the array
+        """    
+        for i in range(len(nums)):
```
Comment (jovan-wong) [inline src/001._two_sum.md]: use index instead of `i`

File: src/001._two_sum.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            j = target - nums[i]
```
Comment (jovan-wong) [inline src/001._two_sum.md]: use `another_addend` instead of `j`

File: src/001._two_sum.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            if j in nums[i+1:]:
+                return [i, nums.index(j,i+1)]
```
Comment (jovan-wong) [inline src/001._two_sum.md]: add a space in the 2 args of `nums.index ` .

**Comment by zmzhouXJTU** [comment]
Much thanks for reviewing my code @stormsunshine @jovan-wong . I will close this PR and pull a new PR

**Pull request closed without merging**